src/index.js

```diff
@@ -454,7 +468,7 @@ const fullWalletFromTicket = async config => {
   const wallet = {
     ticket: displayTicket,
     owner: ownershipNode,
-    voting: votingNode,
+    voting: votingNodes,
```

Comment:
We're fine with this being a bunch of `null`s in most cases?

Reply:
`lodash.filter` handles those dudes -- the array only contains elements for galaxies, and is empty otherwise. :+1:

Reply:
Ah I see. Neat!